### ISSUE TITLE
Web component: Masthead-With custom data - Keyboard navigation - Hamburger menu in 200% zoom text, heading text is missing.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5146]
## Detailed description

Issue:
1. Product & Solution menu:  Paragraph present under Hybrid cloud heading is missing in 200% zoom
2.  paragraph present under Artificial intelligence is missing in 200% zoom

Expected Result:
All the functionality and contents should present in 200% zoom text without loss.

**Comment by NSDrowned** [comment]
Hello @Praveen-sr, I have tested this one thoroughly at 200% zoom but couldn't replicate. Can you please share at what resolution did the problem happen?

![masthead-02](https://user-images.githubusercontent.com/4054756/110956594-66a90380-8310-11eb-87bf-621096d7e6dd.jpg)

![masthead-01](https://user-images.githubusercontent.com/4054756/110956579-627ce600-8310-11eb-845a-d81b8643c85d.jpg)

**Comment by RobertaJHahn** [comment]
Closing based on Praveen's comments above.
